sdk/python/kfp/_client.py

```diff
@@ -294,7 +294,7 @@ def list_pipelines(self, page_token='', page_size=10, sort_by=''):
     """
     return self._pipelines_api.list_pipelines(page_token=page_token, page_size=page_size, sort_by=sort_by)
 
-  def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None):
+  def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None, namespace=None):
```

Comment:
Have you considered to add namespace as part of the client's setting or from env? The namespace is similar with the project id concept in gcloud or gcp sdk.

Reply:
Synced offline. Added TODOs.